### PR TITLE
HZN-662: Create an OSGI-INF blueprint to expose ServiceMonitors as OSGi services

### PR DESCRIPTION
**Pull request by pavanrp1**
Changes as below:
- Added JUnit for the bluprint.xml

### CODE REVIEW

File: features/poller/monitors/core/pom.xml [hunk 1]
```diff
@@ -79,6 +79,204 @@
...
+    </dependency>
+    <dependency>
+      <groupId>org.opennms.dependencies</groupId>
+      <artifactId>castor-dependencies</artifactId>
+      <type>pom</type>
+    </dependency>
+    <dependency>
+      <groupId>org.opennms</groupId>
+      <artifactId>opennms-config</artifactId>
+      <scope>compile</scope>
```
Comment (soleger) [inline features/poller/monitors/core/pom.xml]: I think that all of these added dependencies should be `<scope>test</scope>`. Isn't that correct?

**Comment by soleger** [comment]
I merged some changes from the feature branch into this branch to try and fix the tests. Please review the results in Bamboo and if the tests are still failing, comment on the JIRA issue with what you think still needs to be fixed and then try and make the changes. Thanks!

File: features/poller/monitors/core/pom.xml (hunk 1 above)
Comment (pavanrp1) [inline features/poller/monitors/core/pom.xml]: Yeah this things will be used only in tests. Making it as a scope test.

**Comment by soleger** [comment]
I've merged these changes into the feature branch. Closing PR.

**Pull request closed without merging**